uxarray/grid/coordinates.py

```diff
@@ -329,18 +329,18 @@ def haversine_distance(point1, point2):
     Parameters
     ----------
     point1 : tuple
-        A tuple containing the latitude and longitude of the first point.
+        A tuple containing the lon and lat of the first point.
     point2 : tuple
-        A tuple containing the latitude and longitude of the second point.
+        A tuple containing the lon and lat of the second point.
 
     Returns
     -------
     float
         The distance between the two points on the unit sphere.
     """
     R = 1.0  # Radius of the Earth assumed to be 1 (unit sphere)
```

Comment:
The `R` is never used here. And is not needed as well

Reply:
removed that fun.